packages/pieces/community/google-sheets/src/lib/triggers/new-or-updated-row.trigger.ts

```diff
@@ -107,7 +110,7 @@ export const newOrUpdatedRowTrigger = createTrigger({
     );
 
     const range =
-      triggerColumn === 'all_columns'
+      triggerColumn === ALL_COLUMNS
```

Comment:
What happens when a column is called "all_columns"?
An array-based mechanism might be more appropriate here, then we'd also get multiple trigger columns for free.